cnf-certification-test/identifiers/identifiers.go

```diff
@@ -900,4 +905,11 @@ that there are no changes to the following directories:
 		Remediation:           `If pods are scheduled to realtime kernel nodes, they must add SYS_NICE capability to their spec.`,
 		BestPracticeReference: bestPracticeDocV1dot3URL + " Section 2.7.4",
 	},
+	TestSysPtraceCapabilityIdentifier: {
+		Identifier:            TestSysPtraceCapabilityIdentifier,
+		Type:                  informativeResult,
+		Description:           formDescription(TestSysPtraceCapabilityIdentifier, `check that if process namespace sharing is enabled for a Pod then the SYS_PTRACE capability is allowed`),
+		Remediation:           `allow the SYS_PTRACE capability when enabling process namespace sharing for a Pod`,
```

Comment:
Can this string be added to the remediations.go?  See #349

Reply:
Done.